echoserver/lib/echoserver.ex

```diff
@@ -0,0 +1,19 @@
+defmodule EchoServer do
+  use GenServer
+
+  @moduledoc """
+  Модуль простого эхо сервера.
+  """
+
+  def start_link(opts \\ []) do
+    GenServer.start_link(__MODULE__, [], opts)
+  end
+
+  def init(_) do
+    {:ok, nil}
+  end
+
+  def handle_call(:ping, _from, state) do
+    {:reply, {:pong, node()}, state}
```

Comment:
Это безусловно работает, но в пинг-понг играют одним и тем же шариком. Принимая подачу, игрок отправляет шарик назад ракеткой, а не хватает его рукой, обходит стол, и швыряет в лицо сопернику :)

Я бы выслал обратно сообщение.
```suggestion
  def handle_call(:ping, {pid, _ref}, state) do
    send(pid, {:pong, node()}) # ⇐ THIS
    {:reply, {:pong, node()}, state} # это тоже не повредит
```